ranger/ranger_aws/main.tf

```diff
@@ -72,7 +73,7 @@ resource "aws_security_group" "ec2_sg" {
     from_port       = "${var.ranger_port}"
     to_port         = "${var.ranger_port}"
     protocol        = "tcp"
-    cidr_blocks = ["${var.local_ips}"]
+    cidr_blocks     = "${var.local_ips}"
```

Comment:
@sandeepdabade I just saw this PR. Can you also look at the other open PR and let me know if you still want this to patched in?